docs/repositories/pull-requests.md

```diff
@@ -47,6 +47,17 @@ Click a specific commit to see [detailed information about that commit](commits.
 
 ![Commits tab](images/pull-requests-tab-commits.png)
 
+## Viewing analysis logs
+
+Analysis logs can help you track and understand the performance of the tools and the timing of analyses on your pull requests.
+
+To access these logs, click the **View logs** button in the pull request status area. This opens a modal with two tabs:
```

Comment:
We could add a link to the status area.
Also, replacing **View logs** "button" with "link", for consistency with the content in the [status section](https://github.com/codacy/docs/pull/2133/files#diff-2ddbad93640088a1b84e11fe6135383bd43e8f4b25e29ca5bca803a6c029f4f9R26).

```suggestion
To access these logs, click the **View logs** link in the [pull request status](#status) area. This opens a modal with two tabs:
```